src/components/Pagination.tsx

```diff
@@ -31,6 +36,10 @@ const Pagination: FC<Props> = ({
   totalCount,
   totalPages,
   keyword,
+  pageData: _pageData,
+  itemsPerPage: _itemsPerPage,
+  totalItems: _totalItems,
+  updateSort: _updateSort,
```

Comment:
If we don't use them, maybe we can just avoid adding them here?
```suggestion
```

Reply:
It is nice to be able to spread them like `<Pagination {...pagination} ` as in `InstanceList`.

Reply:
They are not picked up by the divProps which pick up all the props not listed above. That's the reason I added them. Yes they are not used, hence renaming them to the _ prefix.

Reply:
So passing them or not to the props object doesn't change anything, because they're not being used inside the FC anyway. That's why I was proposing the change above.

Reply:
Those 4 props need to be avoided to fall into the divProps, not sure how that would be done otherwise.

Reply:
Ok, I understand the problem we're trying to solve here, but I think this solution is not very robust - ideally, we should use the return type from react-components' `usePagination` to prevent its fields from being passed to the `divProps`. This way, if that ever changes, we don't need to manually add/exclude fields here. However that object doesn't have its dedicated type, so I can't think of a better way to achieve that at the moment - except maybe declaring the type upstream and then update this later.